src/lib.rs

```diff
@@ -42,6 +42,30 @@
 use std::collections::HashMap;
 use testcontainers::{core::WaitFor, Container, Image};
 
+#[derive(Clone, Copy, Debug)]
+pub enum Neo4jLabsPlugin {
```

Comment:
I recommend adding the [`non_exhaustive`](http://doc.rust-lang.org/stable/reference/attributes/type_system.html#the-non_exhaustive-attribute) attribute

```suggestion
#[non_exhaustive]
pub enum Neo4jLabsPlugin {
```

This will allow us to extend this enum without it being a breaking change.